Libs/MRML/Core/vtkMRMLSliceNode.h

```diff
@@ -418,6 +440,11 @@ class VTK_MRML_EXPORT vtkMRMLSliceNode : public vtkMRMLAbstractViewNode
   vtkMatrix4x4 *UVWToSlice;
   vtkMatrix4x4 *UVWToRAS;
 
+  std::map<vtkStdString, vtkMatrix4x4*> OrientationMatrices;
```

Comment:
Very important!! vtkMatrix4x4 should be replaced by vtkSmartPointer<vtkMatrix4x4> otherwise you may leak memory and/or crash. We have to make a copy of the matrix that the caller provides and store that.